osu.Game/Screens/Play/MasterGameplayClockContainer.cs

```diff
@@ -35,8 +35,6 @@ public class MasterGameplayClockContainer : GameplayClockContainer, IBeatSyncPro
         /// </summary>
         public const double MINIMUM_SKIP_TIME = 1000;
 
-        protected Track Track => (Track)SourceClock;
```

Comment:
Out of curiosity mostly, was there a reason this was replaced by soft casts? (see `addSourceClockAdjustments()`)

Reply:
Main goal was to remove as a field to simplify data flow.